views/includes/inputs/address/principal-address-input.html

```diff
@@ -1,3 +1,5 @@
+{% set propertyNameNumberText = "For example, 'Saron House', '116' or 'Unit 37a'" %}
```

Comment:
Just wondering what the value in doing this is? It seems to be only referenced the once, in this file only.

So, why do it for this bit of text and not all the others within the file?

Reply:
Sorry, where's the other place - in which file?

Reply:
👍  thanks. Now I see it. I guess the only thing to be aware of is that if you use that `service-address-include.html` file you need to have the text constants defined in the files where it's being included.